mypy/messages.py

```diff
@@ -870,7 +870,13 @@ def incompatible_argument_note(
             )
             if call:
                 self.note_call(original_caller_type, call, context, code=code)
-
+        if isinstance(callee_type, Instance) and callee_type.type.is_protocol:
+            call = find_member("__call__", callee_type, callee_type, is_operator=True)
+            if call is not None:
+                item = get_proper_type(call)
+                # should we print out all possible overloadings?
```

Comment:
If I'm tracing this right, for overloaded functions it will just print "overloaded function". So I'd say, yes, we should print (and whatever we do, we should probably be consistent with the other branch)

Reply:
I updated it to remove the overloaded check, should now print "overloaded function" consistent with the other note.
Thanks